atomic_reactor/inner.py

```diff
@@ -448,6 +451,28 @@ def __init__(self, source, image, prebuild_plugins=None, prepublish_plugins=None
         if kwargs:
             logger.warning("unprocessed keyword arguments: %s", kwargs)
 
+    def get_orchestrate_build_plugin(self):
+        """
+        Get the orchestrate_build plugin configuration for this workflow
+        if present (will be present for orchestrator, not for worker).
+
+        :return: orchestrate_build plugin configuration dict or None
+        """
+        for plugin in self.buildstep_plugins_conf or []:
+            if plugin['name'] == PLUGIN_BUILD_ORCHESTRATE_KEY:
+                return plugin
+        # Not an orchestrator build
+        return None
```

Comment:
Would it be more Pythonic for this to raise KeyError or something?

Reply:
KeyError felt a bit weird, I made it a ValueError, see if you like it :-) .

Reply:
Looks great, thanks.